### PR TITLE
 filter jenkins header

### PR DESCRIPTION
**Pull request by soulseen**

### CODE REVIEW

**Comment by soulseen** [comment]
@runzexia @zryfish @wansir

File: pkg/models/devops/devops.go [hunk 1]
```diff
@@ -698,6 +698,7 @@ func jenkinsClient(baseUrl string, req *http.Request) ([]byte, http.Header, erro
 	defer resp.Body.Close()
 
 	if resp.StatusCode >= http.StatusBadRequest {
+		log.Errorf("%+v", string(resBody))
```
Comment (runzexia) [inline pkg/models/devops/devops.go]: also return resBody to frontend ?

Comment (runzexia) [reply pkg/models/devops/devops.go]: And do you need to log some context information such as the request URL?

Comment (soulseen) [reply pkg/models/devops/devops.go]: no，just log resBody. return error mssage to frontend.
already log context information in previous func.

Comment (runzexia) [reply pkg/models/devops/devops.go]: At least need to expose http status code to the front end, always 500 is not a good choice

Comment (soulseen) [reply pkg/models/devops/devops.go]: yes，will return resp.StatusCode same as Jenkins returns if StatusCode > 200, not always 500.

File: pkg/models/devops/devops.go [hunk 2]
```diff
@@ -544,8 +545,8 @@ func ToJson(req *http.Request) ([]byte, error) {
 }
 
 func GetNotifyCommit(req *http.Request) ([]byte, error) {
-	baseUrl := fmt.Sprintf(jenkins.Server + GetNotifyCommitUrl + req.URL.RawQuery)
-	log.Infof("Jenkins-url: " + baseUrl)
+	baseUrl := fmt.Sprint(jenkins.Server + GetNotifyCommitUrl + req.URL.RawQuery)
```
Comment (runzexia) [inline pkg/models/devops/devops.go]: why use fmt.Sprint here?

Comment (soulseen) [reply pkg/models/devops/devops.go]: here the var req.URL.RawQuery mabey have some special characters such as "%s", but not should be formart.

Comment (runzexia) [reply pkg/models/devops/devops.go]: Is there a problem with using "+" directly?

Comment (soulseen) [reply pkg/models/devops/devops.go]: It's ok in test.

File: pkg/models/devops/devops.go [hunk 3]
```diff
@@ -68,10 +68,11 @@ func SearchPipelines(req *http.Request) ([]byte, error) {
 }
 
 func SearchPipelineRuns(projectName, pipelineName string, req *http.Request) ([]byte, error) {
-	baseUrl := fmt.Sprintf(jenkins.Server+SearchPipelineRunUrl+req.URL.RawQuery, projectName, pipelineName)
+	baseUrl := fmt.Sprintf(jenkins.Server+SearchPipelineRunUrl, projectName, pipelineName)
+
 	log.Infof("Jenkins-url: " + baseUrl)
```
Comment (runzexia) [inline pkg/models/devops/devops.go]: this log maybe format error

**Pull request merged**